src/controllers/anchor-controller.ts

```diff
@@ -1,6 +1,6 @@
 import { INTERNAL_SERVER_ERROR, OK } from 'http-status-codes';
 import { Request as ExpReq, Response as ExpRes } from 'express';
-import { Logger, Logger as logger } from '@overnightjs/logger';
+import { logger } from "../logger";
```

Comment:
Small but would prefer to keep single vs double quotes consistent

Reply:
done